### ISSUE TITLE
Card flickers/resets in Panel (1 card) view

### ISSUE THREAD

**Issue by AndrewTheWookiee** [Makin-Things/weather-radar-card#30]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser
- [x] I installed using HACS (leave unchecked for manual installs)

**Release with the issue:** 
2.1.1

**Last working release (if known):** 
2.1.1

**Browser and Operating System:** 
Chrome 113.0.5672.127
Firefox 113.0.2
Windows 10

**Card Config (click Show Code Editor and copy paste your config, remove a few of the least significant digits of your lat/lon to avoid posting you exact home address):**

type: custom:weather-radar-card
data_source: RainViewer-DarkSky
map_style: Dark
zoom_level: 6
marker_latitude: 35
marker_longitude: -120
static_map: false
show_marker: true
show_scale: false
show_zoom: false
show_playback: true
show_range: true
square_map: false
show_recenter: true
extra_labels: false
center_longitude: -120
center_latitude: 35
frame_count: 72
frame_delay: 150
restart_delay: 2000

**Description of problem:**
Basically the same issue as https://github.com/Makin-Things/weather-radar-card/issues/22 again.

The card works fine in all other views, and in the single panel view was working fine until somewhat recently. I don't know the exact date or HA version it stopped working in as I haven't used it for a while (no rain/weather to keep track of the past few months) The card flickers at random points, and with each flicker the card increases in height - I'll scroll to the bottom, and after a flicker it will be longer and there is more to scroll down.

Video of issue: https://www.youtube.com/watch?v=FJstMVZf69Q

**Comment by Malvazar** [comment]
Can confirm I'm having the same issue.

**Comment by markfrancisonly** [comment]
Tested the card and it's flashing a couple times a second in my dashboard, have to remove unfortunately

**Comment by HomeWiseAdmin** [comment]
I have the same issue with chrome

**Comment by HomeWiseAdmin** [comment]
It actually does it under all layouts and not only panel card. It flickers and expands the map. Totally unusable.

**Comment by HomeWiseAdmin** [comment]
Confirmed, it actually does under Panel and custom grid. The height just gets increased with every refresh

**Comment by tibmeister** [comment]
Confirmed as well.  Watching this issue.

**Comment by jpettitt** [comment]
Confirmed, it's a weirdness withe hot the card is calculating height.  I'm working on a major refactor of card sizing to handle the sections layout and fixing this will be part of that.
